# STA426 Group3 - Oxford Nanopore


## What is it (Definition from Wikipedia)

Nanopore sequencing uses electrophoresis to transport an unknown sample through an orifice of 10<sup>-9</sup> meters in diameter. A nanopore system always contains an electrolytic solution- when a constant electric field is applied, an electric current can be observed in the system. The magnitude of the electric current density across a nanopore surface depends on the nanoporeâ€™s dimensions and the composition of DNA or RNA that is occupying the nanopore. Sequencing is made possible because, when close enough to nanopores, samples cause characteristic changes in electric current density across nanopore surfaces. The total charge flowing through a nanopore channel is equal to the surface integral of electric current density flux across the nanopore unit normal surfaces between times t1 and t2.

![Device](/Device.png)
![Technology](/Nanopore_Seq.jpg)

## How it works

* Enzyme splits up DNA/RNA, leads one strand in nanopore
* Depending on DNA/RNA-base, different electric current
* By measuring the electric current, it's possible to sequence the DNA/RNA

## Advantage

* Cheap
* Small / Portability
* rapid processing -> display results in real time

## Disadvantage

* High error rate

## Useful links	

* [Wikipedia](https://en.wikipedia.org/wiki/Oxford_Nanopore_Technologies)
* [Oxford Nanopore](https://nanoporetech.com/)
* [Paper on performance of Oxford Nanopore Technology](https://www.sciencedirect.com/science/article/pii/S2214753515000224)
* [A first look](https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.12324)
* [Advantages of long reads for genome assembly](https://nanoporetech.com/sites/default/files/s3/white-papers/WGS_Assembly_white_paper.pdf?submissionGuid=40a7546b-9e51-42e7-bde9-b5ddef3c3512)
